README.md

```diff
@@ -7,9 +7,9 @@ dev-install installs [TripleO standalone](https://docs.openstack.org/project-dep
 dev-install requires that:
 * an appropriate OS has already been installed
 * the machine running dev-install can SSH to the standalone host as either root or a user with passwordless sudo access
-* this machine has Ansible installed, and some dependencies like python3-netaddr.
+* this machine has Ansible installed, and some dependencies like python3-netaddr, python3-jmespath, python3-openstackclient.
 
-For OSP 16.2, the recommended OS is RHEL 8.4. For OSP 17, the recommended RHEL will be 9.
+For OSP 16.2, the recommended OS is RHEL 8.4. For OSP 17.0, the required RHEL will be 9.0.
```

Comment:
17.1 being out soon, I would prefer that we document this one since it's the one that will be supported in long term.

Please change it to 17.1 and 9.2.